lib/remotable/active_record_extender.rb

```diff
@@ -431,7 +430,7 @@ def new_from_remote(remote_resource)
     delegate :local_key,
              :remote_key,
              :remote_model,
-             :remote_attribute_map,
+             :_remote_attribute_map,
```

Comment:
Is there anything outside of this class that would try to call `remote_attribute_map` on the instance? Do we need to have the un-underscored method to maintain the API?

Reply:
Do we need this delegation now if we've aliased our attributes above? Or the renames below?

Reply:
I don't believe we do. Fixed 👍